_posts/2024-05-23-pyos-pyconus-2024-recap.md

```diff
@@ -0,0 +1,258 @@
+“hatchery” to its 6th year at PyCon US. Last year, I had the [privilege of giving
+a talk at the summit](https://www.youtube.com/watch?v=Qxy7bxW72iA). This year, I co-organized, learning the ropes of proposal
+calls, speaker selection, and navigating a summit that filled a room of 80
+people within a conference boasting 2,700 registrations. The prep leading up to
+the summit was manageable, but the days before and the day of were quite the
+adventure!
+
+The summit featured amazing speakers, including:
+
+* Leonard Richardson, who shared
+insights on maintaining core tools like [Beautiful Soup](https://pypi.org/project/beautifulsoup4/) for 20 years and tips on
+handling burnout.
+* [Jessica Tegner](https://github.com/JessicaTegner). Jessica became a maintainer of one of the top 1% of all downloaded packages on PyPI - [pypandoc](https://github.com/JessicaTegner/pypandoc). She talked about her experiences taking over a package with over 3 million monthly downloads. Oh and did I mention that Jessica does all this being visually challenged? The backgrounds (and dedication) of the maintainers in our open source ecosystem always blows my mind.
+
+Working alongside Inessa Pawson, Kara Sowles and Chris Rose who are all also rockstars in the open source world was a fantastic
+experience. Despite some logistical challenges, like the summit filling up
+quickly and a waitlist, the event was a success. Next year (if we are invited back), will be even better as we try to open the doors to include aspiring maintainers and contributors!
+
+{% include video id="L-Ok_89QJOM" provider="youtube" %}
+
+
+Also if you were wondering, yes that Monstera Deliciosa (plant) in the background is real! AND YES it is ginormous! <i class="fa-solid fa-leaf"></i>
+
+## Our second pyOpenSci sprint
+
+This year we had a tremendous turnout of over 20 people from several countries for our 1-day pyConUS sprint. At last count this resulted in about [30 issues and pull requests](https://github.com/orgs/pyOpenSci/projects/12/views/1) where volunteers who were there to support scientists, helped us fix issues on our website, in our tutorials and our packaging and peer review guidebooks. So many long-standing issues and bugs were fixed thanks to this wonderful Python community.
+
+<figure class="half">
+  <img src="{{ site.baseurl }}/images/blog/2024/may/pycon-us-24-pyopensci-sprints-juno.png" alt="Image of the puycon us sprint board where projects signed up for different rooms." />
+  <img src="{{ site.baseurl }}/images/blog/2024/may/pycon-24-sprints-pyopensci.png" alt="Image of the puycon us sprint board where projects signed up for different rooms." />
+  <figcaption>The pyConUS sprint sign-up board had lots of projects. Because the rooms are large projects tend to share spaces. We ended up in the packaging room which was great as it allowed us to do some more difficult work around Python packages with C extensions!</figcaption>
```

Comment:
```suggestion
  <figcaption>The pyConUS sprint sign-up board had lots of projects. Because the rooms are large, projects tend to share spaces. We ended up in the packaging room which was great as it allowed us to do some more difficult work around Python packages with C extensions!</figcaption>
```